eva-accession-import/src/main/java/uk/ac/ebi/eva/accession/dbsnp/listeners/GenericProgressListener.java

```diff
@@ -0,0 +1,95 @@
+/*
+ * Copyright 2014-2018 EMBL - European Bioinformatics Institute
```

Comment:
Please modify your template to write only the current year, otherwise new projects will reflect inaccurate dates.

Reply:
:tada: